src/python/pants/goal/run_tracker_test.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import datetime
+import time
+
+from freezegun import freeze_time
+
+from pants.base.exiter import PANTS_SUCCEEDED_EXIT_CODE
+from pants.goal.run_tracker import RunTracker
+from pants.testutil.option_util import create_options_bootstrapper
+
+
+def test_run_tracker_timing_output() -> None:
+    with freeze_time(datetime.datetime(2020, 1, 1, 12, 0, 0)) as frozen_time:
+        open("BUILDROOT", "w")
```

Comment:
What's this for?

Reply:
This is to trick `RunTracker` into thinking that the sandbox dir of the test process is a pants build root.